data/pom.xml

```diff
@@ -39,12 +39,12 @@
         <module>infinispan</module>
         <module>amazon-s3</module>
         <module>openapi</module>
-        <module>soap</module>
+        <!--module>soap</module-->
         <module>ftp</module>
         <module>odata</module>
         <module>odata4</module>
         <module>sap-gateway</module>
-        <module>util</module>
+        <!--module>util</module-->
```

Comment:
this should not be commented, in the project you can comment the soap dependency out

Reply:
Ok, change applied and the rebuild is successful.